client/client-lib/src/lib.rs

```diff
@@ -1297,9 +1305,18 @@ impl Debug for ClientSecret {
 /// modularized yet but we need the decoding functionality.
 fn module_decode_stubs() -> ModuleDecoderRegistry {
     ModuleDecoderRegistry::new([
-        (MODULE_KEY_LN, Decoder::from_typed(&LightningModuleDecoder)),
-        (MODULE_KEY_WALLET, Decoder::from_typed(&WalletModuleDecoder)),
-        (MODULE_KEY_MINT, Decoder::from_typed(&MintModuleDecoder)),
+        (
```

Comment:
Stuff like this though requires more fundamental changes, outside of scope of this PR.

Reply:
Yep, the client is a mess, needs a rewrite. @mxz42 and I are on it.